nginx/nginx.conf

```diff
@@ -4,11 +4,19 @@ events {
 
 http {
     server {
+        server_name adr;
         listen 80;
         location / {
             proxy_pass http://ckan:5000/;
         }
-        location /giftless {
+        location /giftless/ {
```

Comment:
I think we should use the same "proxy_set" headers for CKAN too (location /)

Reply:
I'm sorry, that's not what I meant, I mean both locations should have the same proxy_set_headers sections, like: 

`
       location / {
           add_header X-Proxy-Cache $upstream_cache_status;
            proxy_set_header Host $host;
            proxy_set_header X-Forwarded-Host $host;
            proxy_set_header X-Forwarded-Prefix /giftless;
            proxy_set_header X-Real-IP $remote_addr;
            proxy_set_header X-Forwarded-Proto $scheme;
            proxy_set_header X-Forwarded-Port $server_port;
            proxy_pass http://ckan:5000/;
        } 
      location /giftless/ {
           add_header X-Proxy-Cache $upstream_cache_status;
            proxy_set_header Host $host;
            proxy_set_header X-Forwarded-Host $host;
            proxy_set_header X-Forwarded-Prefix /giftless;
            proxy_set_header X-Real-IP $remote_addr;
            proxy_set_header X-Forwarded-Proto $scheme;
            proxy_set_header X-Forwarded-Port $server_port;
            proxy_pass http://giftless:6001/;
        }
    }`